mr/src/test/java/org/apache/iceberg/mr/hive/serde/objectinspector/TestIcebergObjectInspector.java

```diff
@@ -90,7 +91,13 @@ public void testIcebergObjectInspector() {
     Assert.assertEquals(3, dateField.getFieldID());
     Assert.assertEquals("date_field", dateField.getFieldName());
     Assert.assertEquals("date comment", dateField.getFieldComment());
-    Assert.assertEquals(IcebergDateObjectInspector.get(), dateField.getFieldObjectInspector());
+    if (MetastoreUtil.hive3PresentOnClasspath()) {
+      Assert.assertEquals(
+              "org.apache.iceberg.mr.hive.serde.objectinspector.IcebergDateObjectInspectorHive3",
+              dateField.getFieldObjectInspector().getClass().getName());
```

Comment:
I'd prefer using the same assertion, but with a different class name. But this is really minor if tests are passing.

Reply:
agree

Reply:
Nit: indentation is off.